tools/cache_perf_tool/main.go

```diff
@@ -167,13 +181,59 @@ outerloop:
 	workerFunc := func() (shouldRetry bool, err error, val interface{}) {
 		return numGoroutines.Load() != int32(0), nil, val
 	}
-	err, _ = base.RetryLoop(parentCtx, "wait for writing goroutines to stop", workerFunc, base.CreateSleeperFunc(200, 100))
+	err, _ = base.RetryLoop(parentCtx, "wait for writing goroutines to stop", workerFunc, base.CreateSleeperFunc(500, 100))
 	if err != nil {
 		log.Printf("Error waiting for stat value (%d) to reach 0: %v", numGoroutines.Load(), err)
 	}
 
 }
 
+func printJavaPropertiesFile(ctx context.Context, dbContext *db.DatabaseContext) {
+	// Print the Java properties file to stdout
+	dbContext.UpdateCalculatedStats(ctx)
+	dbStats := dbContext.DbStats
+	// calculate here avg time to cache seq in ms
+	count := dbStats.Database().DCPCachingCount.Value()
+	timeNano := dbStats.Database().DCPCachingTime.Value()
+	avgTimeNano := float64(timeNano) / float64(count)
+	avgTimeMs := avgTimeNano / 1e6
+	_, _ = fmt.Fprintf(os.Stdout, "high_seq_feed=%d\npending_seq_len=%d\nhigh_seq_stable=%d\ncurrent_skipped_seq_count=%d\nnum_skipped_seqs=%d\n"+
+		"skipped_seq_len=%d\nskipped_seq_cap=%d\ndcp_caching_count=%d\ndcp_caching_time=%d\navg_time_per_seq_ms=%f\n",
+		dbStats.Database().HighSeqFeed.Value(), dbStats.Cache().PendingSeqLen.Value(), dbStats.Cache().HighSeqStable.Value(), dbStats.Cache().NumCurrentSeqsSkipped.Value(),
+		dbStats.Cache().NumSkippedSeqs.Value(), dbStats.Cache().SkippedSeqLen.Value(), dbStats.Cache().SkippedSeqCap.Value(), count,
+		timeNano, avgTimeMs)
```

Comment:
This would be far easier to read and edit if it were one line per stat.

```suggestion
	_, _ = fmt.Fprintf(os.Stdout, "high_seq_feed=%d\n", dbStats.Database().HighSeqFeed.Value())
	_, _ = fmt.Fprintf(os.Stdout, "pending_seq_len=%d\n", dbStats.Cache().PendingSeqLen.Value())
	_, _ = fmt.Fprintf(os.Stdout, "high_seq_stable=%d\n", dbStats.Cache().HighSeqStable.Value())
	_, _ = fmt.Fprintf(os.Stdout, "current_skipped_seq_count=%d\n", dbStats.Cache().NumCurrentSeqsSkipped.Value())
	_, _ = fmt.Fprintf(os.Stdout, "num_skipped_seqs=%d\n", dbStats.Cache().NumSkippedSeqs.Value())
	_, _ = fmt.Fprintf(os.Stdout, "skipped_seq_len=%d\n", dbStats.Cache().SkippedSeqLen.Value())
	_, _ = fmt.Fprintf(os.Stdout, "skipped_seq_cap=%d\n", dbStats.Cache().SkippedSeqCap.Value())
	_, _ = fmt.Fprintf(os.Stdout, "dcp_caching_count=%d\n", count)
	_, _ = fmt.Fprintf(os.Stdout, "dcp_caching_time=%d\n", timeNano)
	_, _ = fmt.Fprintf(os.Stdout, "avg_time_per_seq_ms=%f\n", avgTimeMs)
```